gammapy/irf/io.py

```diff
@@ -1,464 +1,41 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
-from astropy.io import fits
-from astropy.table import Table
 from ..utils.scripts import make_path
-from ..utils.nddata import NDDataArray, BinnedDataAxis
-from ..utils.energy import EnergyBounds
-from .effective_area import EffectiveAreaTable2D, EffectiveAreaTable
+from .effective_area import EffectiveAreaTable2D
 from .background import Background3D
 from .energy_dispersion import EnergyDispersion2D
 from .psf_gauss import EnergyDependentMultiGaussPSF
 
-__all__ = ["CTAIrf", "BgRateTable", "Psf68Table", "SensitivityTable", "CTAPerf"]
+__all__ = ["load_cta_irfs"]
 
 
-class CTAIrf(object):
-    """CTA instrument response function container.
-
-    Class handling CTA instrument response function.
-
-    For now we use the production 2 of the CTA IRF
-    (https://portal.cta-observatory.org/Pages/CTA-Performance.aspx)
-    adapted from the ctools
-    (http://cta.irap.omp.eu/ctools/user_manual/getting_started/response.html).
+def load_cta_irfs(filename = "$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"):
+    """load CTA instrument response function and return a dictionary container.
 
     The IRF format should be compliant with the one discussed
     at http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/.
-    Waiting for a new public production of the CTA IRF,
-    we'll fix the missing pieces.
 
-    This class is similar to `~gammapy.data.DataStoreObservation`,
-    but only contains IRFs (no event data or livetime info).
-    TODO: maybe re-factor code somehow to avoid code duplication.
+    The various IRFs are accessible with the following keys
+    'aeff' is a `~gammapy.irf.EffectiveAreaTable2D`
```

Comment:
This will look much better if you do a bullet list like this (empty line is needed for RST):
```
The various IRFs are accessible with the following keys:

- 'aeff' is a `~gammapy.irf.EffectiveAreaTable2D`
- ...
```
Otherwise it all appears as one wrapped paragraph in the HTML and is harder to read.